package.json

```diff
@@ -49,8 +49,8 @@
   "scripts": {
     "lint": "eslint src && echo '-- linting finished --'",
     "build": "babel src -d dist",
-    "test": "tape -r babel-register -r babel-polyfill src/test",
-    "test:integration": "tape -r babel-register src/test/integration",
+    "test": "tape -r babel-register -r babel-polyfill src/**/*.spec.js",
```

Comment:
This will always run all the tests. What if you want to control which tests are run? You can't just comment/uncomment from an index at will...

Reply:
Besides debugging broken tests, I am unaware of another reason to comment out a test. While using globbing to import tests I would change the name or comment out the code of the test file that I don't want to run until I've finished debugging the issue.

The reason I am thinking this automatic importing is better is because I am finding the maintenance overhead of importing tests manually is becoming a real pain for changing file structure, and during file structure changes I've noticed on occasion 
 that some tests stop getting imported unintentionally.

In my work project I'm finding glob importing to be very helpful in making me more productive. This could just be because I've been struggling with file structure by feature so I may be changing file structures more than normal.

Reply:
I frequently want to run only the current unit's tests by default, and then check the rest for failures only after I've got a solution working for the current unit. I usually do that by adding the file I'm working on at the bottom of the index and commenting out everything else above it, but I'm willing to consider other options. Maybe there's an easier way to select which tests will run and which won't?

Reply:
> I frequently want to run only the current unit's tests by default, and then check the rest for failures only after I've got a solution working for the current unit

I see, I was wondering if you did this some how, it a bit annoying scrolling to the current test I am working on when all the tests are outputting :)

Sounds good, let me give it some thought and I will see if I can come up with something

Reply:
Wouldn't it be cool if there was a test server always running that would let you select which tests are active in a graphical browser window? Something with a tree view that would let you toggle all, none, or some files/folders in the glob hierarchy?

The index file essentially does that for me, but without all the programming work to build the GUI. ;)

Reply:
yeah that would be nice! Without going as far as what you described, I can only think of one thing that might work.

A new test command maybe something like `npm run test:selected`, it would run a test file that supports importing files by an array of globs, we would adjust the file manually while working on a module or feature so it would need to be in the .gitignore with the default checked in.

`npm run watch` would be adjusted to use the new test command

This allows for `npm run test` to always run all the unit tests and would not need to be modified while the `npm run test:selected` becomes the command for working on specific features/modules.

Il see if I can set this up and push it to this pr and see if you like it.

Reply:
The tape glob command is being inconsistant for me again even after updating to 4.8.0.

Its importing less tests than it should and I can't figure out why. I installed the `glob` module and ran a glob require functionality from a js file and it works just fine and imports all the tests correctly.

Going to put this on the back burner while I take care of more important things.